superset/assets/src/explore/components/controls/DateFilterControl.jsx

```diff
@@ -285,9 +308,10 @@ export default class DateFilterControl extends React.Component {
       <Popover id="filter-popover" placement="top" positionTop={0}>
         <div style={{ width: '250px' }}>
           <Tabs
-            defaultActiveKey={this.state.type === TYPES.DEFAULTS ? 1 : 2}
+            defaultActiveKey={this.state.tab === TABS.DEFAULTS ? 1 : 2}
             id="type"
             className="time-filter-tabs"
+            onSelect={() => this.changeTab()}
```

Comment:
```
onSelect={this.changeTab}
```

`this.changeTab` is already `bind` to `this` so can use without wrapping in arrow function